source/bluetooth/MicroBitPartialFlashingService.cpp

```diff
@@ -41,6 +41,11 @@ DEALINGS IN THE SOFTWARE.
 MicroBitPartialFlashingService::MicroBitPartialFlashingService(BLEDevice &_ble, EventModel &_messageBus) :
         ble(_ble), messageBus(_messageBus)
 {
+    packetCount = 0;
+    blockPacketCount = 0;
+    blockNum = 0;
+    offset   = 0;
```

Comment:
Is this necessary for the compiler compatibility?

Reply:
Same as others.